The problem I've hit is this:

```PyTB
Traceback (most recent call last):
  File "/opt/conda/bin/conda-mambabuild", line 10, in <module>
    sys.exit(main())
  File "/opt/conda/lib/python3.10/site-packages/boa/cli/mambabuild.py", line 256, in main
    call_conda_build(action, config)
  File "/opt/conda/lib/python3.10/site-packages/boa/cli/mambabuild.py", line 228, in call_conda_build
    result = api.build(
  File "/opt/conda/lib/python3.10/site-packages/conda_build/api.py", line 180, in build
    return build_tree(
  File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 3078, in build_tree
    packages_from_this = build(metadata, stats,
  File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 2358, in build
    newly_built_packages = bundlers[pkg_type](output_d, m, env, stats)
  File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 1659, in bundle_conda
    files = post_process_files(metadata, initial_files)
  File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 1472, in post_process_files
    post_process(package_name, m.get_value('package/version'),
  File "/opt/conda/lib/python3.10/site-packages/conda_build/post.py", line 289, in post_process
    rm_pyc(files, prefix)
  File "/opt/conda/lib/python3.10/site-packages/conda_build/post.py", line 210, in rm_pyc
    os.unlink(join(prefix, fn))
FileNotFoundError: [Errno 2] No such file or directory: '/home/conda/feedstock_root/build_artifacts/wheel_1683832223026/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/python3.11/site-packages/wheel/__pycache__/__init__.cpython-311.opt-1.pyc'
```

`conda_build/post.py` is trying to unlink the `.pyc`'s that are not getting generated this way. That's curious, though, I think it was supposed to be generated.

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -11,7 +11,7 @@ source:
 build:
   noarch: python
   number: 1
-  script: {{ PYTHON }} -c "import flit_core.buildapi; flit_core.buildapi.build_wheel('.')" && {{ PYTHON }} -m installer wheel-*.whl
+  script: {{ PYTHON }} -c "import flit_core.buildapi; flit_core.buildapi.build_wheel('.')" && {{ PYTHON }} -m installer wheel-*.whl && ls -Rl $(python -c 'import sysconfig; print(sysconfig.get_paths()["purelib"])')/wheel
   entry_points:
     - wheel = wheel.cli:main
 

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -11,7 +11,7 @@ source:
 build:
   noarch: python
   number: 1
-  script: {{ PYTHON }} -c "import flit_core.buildapi; flit_core.buildapi.build_wheel('.')" && {{ PYTHON }} -m installer --no-compile-bytecode wheel-*.whl && ls -Rl $(python -c 'import sysconfig; print(sysconfig.get_paths()["purelib"])')/wheel
+  script: {{ PYTHON }} -c "import flit_core.buildapi; flit_core.buildapi.build_wheel('.')" && {{ PYTHON }} -m installer --no-compile-bytecode wheel-*.whl
   entry_points:
     - wheel = wheel.cli:main
 

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -11,7 +11,7 @@ source:
 build:
   noarch: python
   number: 1
-  script: {{ PYTHON }} -c "import flit_core.buildapi; flit_core.buildapi.build_wheel('.')" && {{ PYTHON }} -m installer wheel-*.whl && ls -Rl $(python -c 'import sysconfig; print(sysconfig.get_paths()["purelib"])')/wheel
+  script: {{ PYTHON }} -c "import flit_core.buildapi; flit_core.buildapi.build_wheel('.')" && {{ PYTHON }} -m installer --no-compile-bytecode wheel-*.whl && ls -Rl $(python -c 'import sysconfig; print(sysconfig.get_paths()["purelib"])')/wheel
   entry_points:
     - wheel = wheel.cli:main
 

